content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -0,0 +1,278 @@
+kind: documentation
+aliases:
+- /tagging/unified_service_tagging
+further_reading:
+- link: "/tagging/using_tags"
+  tag: "Documentation"
+  text: "Learn how to use tags in the Datadog UI"
+- link: "/blog/autodiscovery-docker-monitoring"
+  tag: "Blog"
+  text: "Learn more about Autodiscovery"
+---
+
+## Overview
+Unified service tagging ties Datadog telemetry together through the use of three [reserved tags][1]: `env`, `service`, and `version`. These tags allow you to correlate related logs and traces by version of a service, trace a spike in CPU usage to corresponding areas of a service, and organize infrastructure more effectively.
+
+The [environment variables][2] `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` form a single point of configuration for all telemetry emitted directly from your service's runtime: [traces][3], [logs][4], and [StatsD metrics][5]. These variables are specific to the runtime of your service. They allow you to define core tagging configuration within your application, rather than across your application and the Agent.
+
+When spans are configured with `DD_VERSION`, the tracer will add version to all spans that fall under the service that "owns" the tracer (generally DD_SERVICE). This means that if your service creates spans with the name of an external service, those spans will not receive version as a tag. As long as version is present in spans, it will be added to trace metrics generated from those spans. The version can be added manually in-code or automatically by the tracer.
+
+When available, version is added to logs like any other tag.
+
+At the very least they will be used by the APM and [Dogstatsd clients][6] to tag trace data and statsd metrics with `env`, `service`, and `version`. If enabled, the APM tracer will also inject the values of these variables into your logs.
+
+Depending on your CI/CD pipeline and how you run your service (e.g., as a binary or through a Docker container), you may wish to set these environment variables depending on whether your environment is [containerized](#containerized-environment) or [non-containerized](#non-containerized-environment).
+
+{{< img src="tagging/unified_service_tagging/overview.gif" alt="Unified Service Tagging"  >}}
+
+## Requirements
+
+- Unified service tagging requires knowledge of tagging and assigning tags. If you are unsure of how to configure tags, read the [Getting Started with Tagging][7] guide and [Assigning tags][8] documentation before proceeding to configuration.
+
+- Unified service tagging requires setup of the [Datadog Agent][9].
+
+- This guide requires [Autodiscovery][10] for unified service tagging in a containerized environment.
+
+## Recommendations
+
+If you're using [connected Logs and Traces][11], it is recommended to enable automatic logs injection. The APM tracer will then automatically add these fields to your logs, thereby eliminating manual configuration for those fields elsewhere.
+
+## Configuration
+
+### Containerized environment
```

Comment:
Should we use tabs to split between containerised and non-containerised installations?

(Not sure if tabs allow to use section within them